config/config.go

```diff
@@ -55,6 +61,27 @@ func (d DatadogConfig) IsDataDogConfigValid() bool {
 	return true
 }
 
+// Validate the DatadogConfig. Returns the first error found, returns nil if
+// the configuration is good.
+func (d DatadogConfig) Validate() error {
+	if d.Env == "" {
+		return errors.New("DD_ENV must be defined")
+	}
+	if d.Service == "" {
+		return errors.New("DD_SERVICE must be defined")
+	}
+	if d.ServiceVersion == "" {
+		return errors.New("DD_VERSION must be defined")
+	}
+
+	// DSD or APM must be configured`
```

Comment:
No need to copy the trailing backtick from IsDataDogConfigValid :)

Reply:
lol, I'll clean that up. No need to copy the comment as the error describes the same

Reply:
removed